docs/UsersGuide.asciidoc

```diff
@@ -1021,6 +1021,28 @@ by adding additional query parameters, e.g.:
   `openSUSE Leap 15 ARM`. This applies to parameters for filtering job settings
   as well.
 
+=== Finding tests by quering particular job settings
+
+In case you are looking for a way to get a list of all the tests based on their
+settings, here two ways to achieve it.
+
+[source,sh]
+----
+curl -s "https://openqa.suse.de/api/v1/job_settings/jobs?key=*_TEST_ISSUES&list_value=39911"
```

Comment:
You are using a lot of `--osd` / openqa.suse.de in the examples you added.
We should prefer to use public urls.
Also I don't see why we should add both the curl and the openqa-cli usage here.
If we do that for every API call, the docs will get noisy.
Just describe the `openqa-cli` call. And leave off the `--osd` switch.